pkg/test/dualstack/cloud_providers_dualstack_test.go

```diff
@@ -176,6 +177,34 @@ func TestCloudClusterIPFamily(t *testing.T) {
 			skipNodes:           true,
 			skipHostNetworkPods: true,
 		},
+		{
+			cloudName: "openstack",
+			osNames: []string{
+				// "centos",
+				// "flatcar",
+				// "rhel",
+				// "sles",
+				"ubuntu", // others not tested
+			},
+			cni:                 "cilium",
+			ipFamily:            util.DualStack,
+			skipNodes:           true,
+			skipHostNetworkPods: true,
+		},
+		{
+			cloudName: "openstack",
+			osNames: []string{
+				"centos",
+				// "flatcar",
+				// "rhel",
+				// "sles",
```

Comment:
Please remove these comments, you should add them back once they are implemented. Not sure if there is any value in having commented out values here.